source/includes/common-installation.rst

```diff
@@ -242,11 +242,11 @@ For example:
 
    $ nano /etc/fstab
 
-     # <file system>  <mount point>  <type>  <options>         <dump>  <pass>
-     LABEL=DISK1      /mnt/disk1     xfs     defaults,noatime  0       2
-     LABEL=DISK2      /mnt/disk2     xfs     defaults,noatime  0       2
-     LABEL=DISK3      /mnt/disk3     xfs     defaults,noatime  0       2
-     LABEL=DISK4      /mnt/disk4     xfs     defaults,noatime  0       2
+     # <file system>  <mount point>  <type>  <options>                <dump>  <pass>
+     LABEL=DISK1      /mnt/disk1     xfs     defaults,noatime,nofail  0       2
```

Comment:
Why is this being documented here `nofail` ?

Reply:
It was a request from @raghav-vk in #874.

Reply:
yeah but it must not be default recommendation, this can lead to problems @djwfyi it will ignore the mounts 

```
           Do not report errors for this device if it does not exist.
```

nofail is not what we want in normal circumstances.

Reply:
That should be fine if we don't need it by default. Probably a NOTE: When running on the cloud, the instances configured without `nofail` can become inaccessible. with no-fail, we can access the instances and rectify mount points. This was the case with couple of customers on AWS

Reply:
Adding notes to Raghav's points and removing `nofail` in the examples.